admin/config.js

```diff
@@ -27,7 +27,8 @@ module.exports = {
   Data: {
     allowedWords: process.env.ALLOWED_WORDS || 'aaa,bcd,dcd,tfg,bxx',
     pinSubmissionMaxAttempts: process.env.PIN_SUBMISSION_MAX_ATTEMPTS || 100,
-    helplineNumber: process.env.HELPLINE_NUMBER || '0345 278 8080'
+    helplineNumber: process.env.HELPLINE_NUMBER || '0345 278 8080',
+    pupilCensusMaxSize: process.env.PUPIL_CENSUS_MAX_SIZE || 100 * 1024 * 1024
```

Comment:
What are the units?  I'd suggest calling this `PUPIL_CENSUS_MAX_FILE_UPLOAD_MB` if that's what it is

Reply:
@jon-shipley updated on that